sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/statsbeat/_statsbeat_metrics.py

```diff
@@ -197,12 +204,22 @@ def _get_feature_metric(self, options: CallbackOptions) -> Iterable[Observation]
         # Check if it is time to observe long interval metrics
         if not self._meets_long_interval_threshold(_FEATURE_METRIC_NAME[0]):
             return observations
+        # Feature metric
         # Don't send observation if no features enabled
-        if self._feature is _StatsbeatFeature.NONE:
-            return observations
-        attributes = dict(_StatsbeatMetrics._COMMON_ATTRIBUTES)
-        attributes.update(_StatsbeatMetrics._FEATURE_ATTRIBUTES)
-        observations.append(Observation(1, dict(attributes)))
+        if self._feature is not _StatsbeatFeature.NONE:
+            attributes = dict(_StatsbeatMetrics._COMMON_ATTRIBUTES)
+            attributes.update(_StatsbeatMetrics._FEATURE_ATTRIBUTES)
+            observations.append(Observation(1, dict(attributes)))
+
+        # instrumentation metric
+        # Don't send observation if no instrumentations enabled
+        instrumentations = _utils.get_instrumentations()
+        if instrumentations != 0:
```

Comment:
Similar here. If we could name this something like "instrumentation_bits" or "instrumentation_bit_map", I think it would be clearer.